recipes/itk/all/conanfile.py

```diff
@@ -106,38 +106,33 @@ def generate(self):
         tc.variables["BUILD_DOCUMENTATION"] = False
         tc.variables["ITK_SKIP_PATH_LENGTH_CHECKS"] = True
 
-        tc.variables["ITK_USE_SYSTEM_LIBRARIES"] = True
         tc.variables["ITK_USE_SYSTEM_DCMTK"] = True
         tc.variables["ITK_USE_SYSTEM_DOUBLECONVERSION"] = True
         tc.variables["ITK_USE_SYSTEM_EIGEN"] = True
         tc.variables["ITK_USE_SYSTEM_FFTW"] = True
         tc.variables["ITK_USE_SYSTEM_GDCM"] = True
         tc.variables["ITK_USE_SYSTEM_HDF5"] = True
-        tc.variables["ITK_USE_SYSTEM_ICU"] = True
```

Comment:
Why does ICU is no longer listed? I still see it available for 5.1.2: https://github.com/InsightSoftwareConsortium/ITK/blob/v5.1.2/Modules/ThirdParty/DCMTK/CMakeLists.txt#L11